File: calculate_average.c
void main() {
    int arr[] = {85, 90, 89, 92, 78};
    int n = 5;
    int sum = 0;

    for (int i = 0; i < n; i++) {
        sum += arr[i];
    }

    float average = (float)sum / n;

}

File: calculate_average.rs
fn main() {
    let arr = [85, 90, 89, 92, 78];
    let n = 5;
    let mut sum = 0;

    for i in 0..n {
        sum += arr[i];
    }

    let average = sum as f32 / n as f32;
}
